### DISCUSSION TITLE
Input with "gum" ... a bug?

### DISCUSSION THREAD

**Discussion by moelholm** [google/zx#545] [Help needed]
Google ZX bug? 🧐 Try this:

    await $`gum input`

Input *MOVES*  in a steady pace to the right? Watch it take off to the right ... attempting to escape the screen 🤯:

<img width="443" alt="image" src="https://user-images.githubusercontent.com/8393156/199801610-09aa3e6b-1c40-4733-a39c-605624122701.png">

I barely managed to capture this 🤣. 

_*(Problem within both zsh and fish. On Mac OS. In iTerm2. Not a problem with raw BASH…)_

**Comment by antonmedv** [comment]
TUI is not supported by zx. gum is using bubbletea lib which is TUI.
